kolibri_explore_plugin/assets/src/views/SearchPage.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 
-  <div class="d-flex flex-column h-100 search-page">
```

Comment:
There is a helper in bootstrap for min-height 100vh: `min-vh-100`.
https://getbootstrap.com/docs/5.0/utilities/sizing/#relative-to-the-viewport

Reply:
Thanks! I'll use that instead of the manual css.